### DISCUSSION TITLE
How to implement a searchbox like on the AntD website?

### DISCUSSION THREAD

**Discussion by igalci** [ant-design/ant-design#39601] [Ideas]
If you go on the ant.design official website, you will notice that their main search box has text and icon on the left, and a smaller sub text under each item.

What is that component called? I looked in the autocomplete page and haven't found anything similar...

**Comment by SignDawn** [comment]
https://ant.design/components/input-cn

**Reply by igalci** [reply]
incorrect. It does not feature a search box with autosuggestions like on the official ant design website.

**Comment by BoyYangzai** [comment]
may you can try [DocSearch](https://docsearch.algolia.com/)
It is used in `AntV`

**Reply by igalci** [reply]
Yes you are right, they are not using AntD for the search box. Thank you for pointing it out.

**Comment by meng1204** [comment]
any solutions?
